### PR TITLE
Set title, breadcrumb, and path for pages without drupal nodes

### PR DESCRIPTION
**Pull request by beeyayjay**
## Description
Adds a helper function to update the title and entityUrl values for generated pages, e.g. listing pages.

Also updates paginatePages() to use the entityUrl on the page object, since that's now correct.

### CODE REVIEW

File: src/site/stages/build/drupal/metalsmith-drupal.js [hunk 1]
```diff
@@ -105,29 +97,48 @@ function paginatePages(
...
+  const generatedPage = Object.assign({}, page);
+  generatedPage.entityUrl.breadcrumb.push({
+    url: { path: drupalPagePath },
+    text: page.title,
+  });
+  generatedPage.entityUrl.path = `${drupalPagePath}/${pathSuffix}`;
+  generatedPage.title = title;
+  return generatedPage;
+}
+
```
Comment (ncksllvn) [inline src/site/stages/build/drupal/metalsmith-drupal.js]: Is this only for press releases, or is it planned to be used for other pages as well?

**Review by ncksllvn — COMMENTED** [review]
Looks good! `createEntityUrl` sounds a little generic so just wondering if it should be renamed to indicate press releases-only, or if there's plan to use it elsewhere.

File: src/site/stages/build/drupal/metalsmith-drupal.js (hunk 1 above)
Comment (acabouet) [reply src/site/stages/build/drupal/metalsmith-drupal.js]: We're gonna use it on other pages as well. Anything that doesn't have a corresponding page entity in Drupal - the events listing page, the press release listing page, the services pages, etc.

**Review by acabouet — APPROVED** [review]
Looks great

**Pull request merged**